tests/virt/upgrade/conftest.py

```diff
@@ -192,7 +196,7 @@ def vm_from_template(
     cpu_model,
     template_labels,
     networks=None,
-    run_strategy=None,
+    run_strategy=VirtualMachine.RunStrategy.HALTED,
```

Comment:
why is it needed? template already is set to halted

Reply:
we use `vm_from_template` fuction for tests about run_strategy, so they pass `run_strategy` as parameter

when we set `run_strategy=None` - it overrides what we have in templates and VM is not created

Reply:
so it seems it's not needed as [HALTED is also default for VirtualMachineForTestsFromTemplate](https://github.com/RedHatQE/openshift-virtualization-tests/blob/main/utilities/virt.py#L1098)

Reply:
this function is used in several places, for example:
https://github.com/RedHatQE/openshift-virtualization-tests/blob/main/tests/virt/upgrade/conftest.py#L249

so we have to set some values to run_strategy here